pkgs/applications/misc/easystroke/default.nix

```
@@ -0,0 +1,31 @@
+# based on easystroke-git from Arch AUR
+# as found on https://github.com/teu5us/easystroke-nix
+
+{ pkgs, ... }:
+
+with pkgs;
+
+stdenv.mkDerivation rec {
+  name = "easystroke";
+  version = "patched-2018-12-05";
```

```suggestion
  version = "0.6.0";
```
packages should be as close as possible to upstream vanilla.